python-threatexchange/threatexchange/fetcher/fetch_state.py

```diff
@@ -61,10 +148,27 @@ class FetchedSignalDataBase:
     will need to store that here.
     """
 
-    opinions: t.List[SignalOpinion]
+    def get_as_opinions(self) -> t.List[SignalOpinion]:
+        return [SignalOpinion.get_trivial()]
+
+    @classmethod
+    def merge_metadata(
+        cls, _older: "FetchedSignalMetadata", newer: "FetchedSignalMetadata"
+    ) -> "FetchedSignalMetadata":
+        """
+        The merge strategy when streaming updates.
+        """
+        return newer
```

Comment:
Only if newer.owner == older.ower, right? 

Otherwise, we should not try to merge them?

Or prefer our own owner_id?

Reply:
I see that this is done as part of SimpleFetchedSignalMetadata, so feel free to ignore.

Reply:
Nope, it really means completely replace without checking. However, the docstring is wrong